.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve our libraries
+
+---
+
+<!--
+  Thanks for participating in this project! We will try to answer as soon as possible 🙏
+-->
+
+**Describe the bug 🐛**
+A clear and concise description of what the bug is.
+
+**To Reproduce 🔍**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior 💭**
+A clear and concise description of what you expected to happen.
+

```

Comment:
Just suggesting a new option, but I'm really fine with how it is right now.

We can put a section `Actual behavior` as well. That means that step 4 in the "to Reproduce" section would be removed.